JS/Lesson31/gulcin/todo-app/src/exercise-components/Exercise9/index.jsx

```diff
@@ -0,0 +1,14 @@
+export const Ex9 = () => {
+   
+   function changeColor (e) {
+        e.target.style.backgroundColor = e.target.innerText;
```

Comment:
In React, we avoid interacting with DOM directly (e.target.style.backgroundColor). Instead, you can solve it using inline styles or adding a class depending on which button is clicked, and add CSS code to change background color depending on which class is added.
```
import { useState } from 'react';

export const Ex9 = () => {
	const [color, setColor] = useState('#FFF');

	function changeColor(color) {
		setColor(color);
	}

	return (
		<div style={`backgroundColor: ${color}`}>
			<button onClick={() => changeColor('#FF0000')}>Red</button>
			<button onClick={() => changeColor('#0000FF')}>Blue</button>
			<button onClick={() => changeColor('#A020F0')}>Purple</button>
			<button onClick={() => changeColor('#008000')}>Green</button>
		</div>
	);
};
```

Reply:
I changed the bg color of the div but you can do the same for the background of the buttons, just it will take a bit more time.

Reply:
Can you check again? Fixed.